hazelcast-client/src/main/java/com/hazelcast/client/connection/ClientConnectionManager.java

```diff
@@ -26,25 +26,32 @@
 import java.util.Collection;
 
 /**
- * Responsible for managing {@link com.hazelcast.client.connection.nio.ClientConnection} objects.
+ * Responsible for managing {@link com.hazelcast.client.connection.nio.ClientConnection} instances.
  */
 public interface ClientConnectionManager extends ConnectionListenable {
 
     /**
-     * Check if client connection manager is alive.
+     * Check this ClientConnectionManager is alive.
      * ClientConnectionManager is not alive only when client is closing.
      *
      * @return true if alive, false otherwise.
      */
     boolean isAlive();
 
     /**
+     * Gets the Connection to the given member address.
+     *
+     * If address is null, null is returned.
+     *
      * @param address to be connected
      * @return connection if available, null otherwise
      */
     Connection getActiveConnection(Address address);
 
     /**
+     * Gets the Connection to the given member address if there already is a connection, or
+     * otherwise triggers the connection asynchronously and then null is returned.
+     *
      * @param address to be connected
      * @return associated connection if available, creates new connection otherwise
```

Comment:
creates new connection and returns null

Reply:
This doesn't make a lot of sense. 

The javadoc in this PR makes it clear what is happening. Either the found connection is returned, or asynchronously the connection is made and null is returned.